halo/attest/keeper/keeper.go

```diff
@@ -51,7 +50,7 @@ type Keeper struct {
 	voteExtLimit   uint64
 	trimLag        uint64
 
-	valCache *valCache
+	valAddrs *valAddrs
```

Comment:
nit: i'd still name the field and the type `valAddrsCache`, just so it's obvious what it's doing